### PR TITLE
Complete num msgs

### PR DESCRIPTION
**Pull request by hunt0r**
[NOTE]
Dependent on #33 and appears dependent on #41, but actually could stand apart from that one.

[Description]
A very simple update: Just set the number of log messages (obj.numMsgs) equal to the last log line number. (LineNo)

### CODE REVIEW

File: Ardupilog.m [hunk 1]
```diff
@@ -157,6 +157,8 @@ function delete(obj)
             % Construct the LineNo for the whole log
             LineNo_ndx_vec = sort(vertcat(obj.valid_msgheader_cell{:,2}));
             LineNo_vec = [1:length(LineNo_ndx_vec)]';
+            % Save the last LineNo as the number of messages in the log
+            obj.numMsgs = LineNo_vec(end);
```
Comment (Georacer) [inline Ardupilog.m:164]: I was wondering about this: Should numMsgs contain the total number of msgs in the log, or just the amount of filtered messages?
What would be useful to you?

Comment (hunt0r) [reply Ardupilog.m:164]: I don't feel strongly either way. Unless you do, I guess I propose we split into 2 clearer fields: numLogMsgs and numFiltMsgs.

I like the idea of capturing the total number of log messages somewhere in the Ardupilog properties to go alongside the .bin filename, pathname, and other related info. Let's call this "numLogMsgs." If no filtering is used, this should be equal to the total number of messages in the Ardupilog object.

How about if we want other property(s) to capture filtered-log info, we create those separately? For example, if we use message filtering, how about we could make the msgFilter public? That would tell someone looking at the log that a filter was used. We could also create a property "numFiltMsgs" that tells how many messages total are in the (filtered) Ardupilog object.

Comment (Georacer) [reply Ardupilog.m:164]: Yeah, these steps sound ideal:
1. Add totalNumMsgs
2. Keep numMsgs
3. Add the used message filter

Will you push an update commit? (remember to pull the branch locally, as I did a rebase and re-wrote history for this branch)

**Comment by hunt0r** [comment]
Per our discussion, I'm closing this and opening a new one.

**Pull request closed without merging**